osm2streets-java/StreetNetwork.java

```diff
@@ -0,0 +1,24 @@
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+class StreetNetwork {
+	static {
+		System.loadLibrary("osm2streets_java");
```

Comment:
You may need to prepend `lib` here. I had to fiddle around to get this to work. The actual file is `libosm2streets_java.so` for me